server.py

```diff
@@ -1,17 +1,38 @@
-from flask import Flask, request, jsonify, g
-from cryptography import exceptions
-from decrypter import Decrypter
-from structlog import wrap_logger
 import binascii
 import logging
 import os
 
+import yaml
+from cryptography import exceptions
+from flask import Flask, request, jsonify
+from sdc.crypto.secrets import SecretStore, validate_required_secrets
+from sdc.crypto.invalid_token_exception import InvalidTokenException
 from sdx.common.logger_config import logger_initial_config
+from structlog import wrap_logger
+
+import settings
+import sdc.crypto.decrypter
+
+EXPECTED_SECRETS = [
+    "SDX_SECRET_KEY",
+]
+
+KEY_PURPOSE_SUBMISSION = 'eq-submission'
 
 
 __version__ = "1.2.0"
 
+
 app = Flask(__name__)
```

Comment:
At some point we really need to start generating apps via app factories http://flask.pocoo.org/docs/0.12/patterns/appfactories/